doc/conf.py

```diff
@@ -53,12 +56,29 @@
     "sphinxcontrib.napoleon",
     "nbsphinx",
     "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
     "sphinx.ext.mathjax",
     "sphinxcontrib.bibtex",
     "sphinx_copybutton",
     "sphinx_asdf",
+    "numpydoc",
 ]
 
+# autosummary --------------------------------------------------------------------------
+autosummary_generate = True
+numpydoc_show_class_members = False
+
+# version = .__version__
+# The full version, including alpha/beta/rc tags.
+release = numpydoc.__version__
```

Comment:
This should give the `numpydoc` instead of `weldx` version? (see line 43)

Reply:
I have to review the whole block (5 lines) myself. I am not even sure if it is needed.  I will try removing it and see what happens :)

Reply:
Just checked the numpydoc documentation and updated the block (removed the line you mentioned)